src/main/java/com/gregtechceu/gtceu/api/mui/drawable/UITexture.java

```diff
@@ -302,14 +303,28 @@ public Builder fullImage() {
         }
 
         /**
-         * Specify a sub area of the image in pixels.
+         * Specify a sub area of the image in pixels, with a position and a size
          *
          * @param x x in pixels
          * @param y y in pixels
          * @param w width in pixels
          * @param h height in pixels
+         * @see #subAreaXYWH(int, int, int, int)
          */
+        @ApiStatus.Obsolete
         public Builder xy(int x, int y, int w, int h) {
```

Comment:
You could just remove this instead of marking it obsolete, you don't need to worry about maintaining the mui API considering it isn't released yet.